praw/models/redditors.py

```diff
@@ -50,3 +54,33 @@ def stream(self, **stream_options):
         :returns Redditor profiles, which are a type of :class:`.Subreddit`.
         """
         return stream_generator(self.new, **stream_options)
+
+    def user_data_by_account_ids(self, ids):
+        """Get user data by account IDs.
+
+        :param ids: A list of redditor fullnames.
+        :returns: A dictionary that mapping redditor fullnames to dictionaries
```

Comment:
To be honest I was at a bit of a conflict in writing this patch. I did have *some* good reasons for this dictionary approach: I went into this intending this method to be a simple low-level wrapper around the end point (hence reuse of the long method name), everyone knows how to use dictionaries, and I’m not really a fan of the massively dynamic lazy-load nature of those PRAW objects.

Since this is a bulk processing thing, returning a dictionary really is inappropriate. I’ve fixed these problems now and I’ve made it yield proper objects.